src/clj/puppetlabs/services/jruby_pool_manager/jruby_core.clj

```diff
@@ -145,6 +149,7 @@
   [config :- {schema/Keyword schema/Any}]
   (-> config
       (update-in [:compile-mode] #(keyword (or % default-jruby-compile-mode)))
+      (update-in [:compat-version] #(str (or % default-jruby-compat-version)))
```

Comment:
Huh. I feel like hocon shouldn't convert floats to ints if they've got the decimal point. Maybe that's a bug?

I tried it out with the fancy new hocon cli too, it's the same there
```
echo "foo: 2.0" | ./bin/hocon get foo
2
```